torch/csrc/dynamo/eval_frame.c

```diff
@@ -1,6 +1,7 @@
 #define PY_SSIZE_T_CLEAN
 #include <torch/csrc/dynamo/cpython_defs.h>
 #include <torch/csrc/utils/python_compat.h>
+#include <torch/csrc/utils/pythoncapi_compat.h>
```

Comment:
This should be coming from python_compat.h already.
Not sure if we prefer to re-include or not?